content/chat/rooms/messages.textile

```diff
@@ -360,10 +417,26 @@ const MyComponent = () => {
 };
 ```
 
+```[swift]
+let messageToDelete: Message
+let deletedMessage = try await room().messages.delete(
+  message: messageToDelete,
+  params: .init(description: "Message deleted by user")
+)
+```
+
+```[kotlin]
+val messageToDelete: Message
+val deletedMessage = room().messages.delete(
+  messageToDelete,
+  operationDescription = "Message deleted by user",
+)
+```
+
 h3(#filter-deletes). Filter for deletes
 
-blang[javascript].
-  Use the "@messages.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Messages.html#subscribe method to receive messages in a room. To filter for deleted messages, provide a listener that checks the @type@ property of the message event:
+blang[javascript,swift,kotlin].
+  Use the <span lang="javascript">"@messages.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Messages.html#subscribe</span><span lang="swift">"@messages.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-swift/main/AblyChat/documentation/ablychat/messages/subscribe()-8jolq</span><span lang="kotlin">"@messages.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-kotlin/main/dokka/chat-android/com.ably.chat/-messages/subscribe.html</span> method to receive messages in a room. To filter for deleted messages, provide a listener that checks the <span lang="javascript">@type@</span><span lang="swift">@action@</span><span lang="kotlin">@type@</span> property of the message event:
```

Comment:
We get some odd formatting here with `8jolq` on the end

Reply:
Fixed